tools/wheel/image/provision-python.sh

```diff
@@ -50,5 +50,7 @@ pip install \
     auditwheel
 
 if [[ "$(uname)" == "Linux" ]]; then
-    pip install patchelf
+    # TODO(19261): on 2023-04-24 the release of patchelf==0.18.0.0 results in
+    # misaligned sections.  For now we pin to the most recent working version.
```

Comment:
A link to the upstream issue may be useful (though it's probably in that issue mentioned in the `TODO`).